### ISSUE TITLE
Vertical ToC cannot be accessed via keyboard

### ISSUE THREAD

**Issue by szczz** [ramp4-pcar4/storylines#560]
## Description

Reported in the 4/29/2025 RESPECT Testing Party. Currently, there is no way to access the individual table of contents items when we we're in vertical mode. This is an accessibility issue that we'll need to fix asap. 

>Technically a storylines thing, but I can’t figure out how to get keyboard access to the Chapters / Table of Contents. I can get into the thing that toggles the view to show text or just show hexagons. But cannot select a hexagon to zoomie to that section.

**Comment by spencerwahl** [comment]
This looks like it is only an issue with the editor's preview. Normal storylines should be all good.
